src/wai_data_tools/manual_labeling.py

```diff
@@ -61,16 +62,16 @@ def prev(self, event):
 
     def draw_img(self):
         """Handle showing images."""
-        self.plt_img.set_array(self.frame_dict[self.index]["img"])
+        self.plt_img.set_array(self.frame_dict[self.index]["image"])
         self.ax_img.set_title(f"Frame {self.index}")
         self.ax_togg.set_title(f"Class: {self.frame_dict[self.index]['target']}")
         plt.pause(0.001)
 
-    def toggle_label(self, event):
+    def toggle_label(self, _=None):
```

Comment:
another option instead of having '_' you can use '_content', then the next person would at least know what the input should have been there, or what they can use from that context

Reply:
Good point, will fix